context/requirements.txt

```diff
@@ -5,6 +5,6 @@ requests==2.22.0
 pyyaml==5.4
 python-datauri==0.2.8
 python-frontmatter==0.5.0
-vitessce==0.1.0a9
+vitessce==0.1.0a11
```

Comment:
What's the relationship between the vitessce python  and the vitessce js version numbers? Down the road, maybe it would make sense for the version numbers to match?

Reply:
I think this time is a coincidence that the numbers are even close, but we can try for that maybe.  There are fairly independent large development pushes in the respective libraries though.